SportShoesStore.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include <string>
+#include "Sneakers.h"
+#include <vector>
+using namespace std;
+
+class SportShoesStore {
+private:
+	vector<Sneakers> s;
+	string brand[5] = { "Adidas","Nike","Puma","UnderUrmor","NewBalace" };
```

Comment:
Please don't embarrass yourself and change brand names to UnderArmour and NewBalance :)